src/html/ui/user/settings.tpl.html

```diff
@@ -75,6 +75,17 @@
         <div class="side-column"><a ng-click="leaveCommunity(membership.community_id, $index)">Leave</a></div>
       </div>
     </div>
+    <div class="section" collapse="!expand2">
+      <div ng-show="!user.memberships.length">
+        <div class="setting-item">
+          <div class="full-column">
+            <div ng-controller="NavCtrl">
```

Comment:
i should have caught this in the prior PR: rather than using NavCtrl here, it'd be better to pull out the joinCommunity method into a service and then have both NavCtrl and this view's existing controller call it.